README.md

```diff
@@ -120,7 +125,7 @@ Examples of `Build` resource using the example strategies installed by default.
       name: buildpacks-v3
       kind: ClusterBuildStrategy
     output:
-      image: <MY_REGISTRY>/<MY_USERNAME>/<MY_REPO>:latest
+      image: $MY_REGISTRY/$MY_USERNAME/$MY_REPO:latest
```

Comment:
On my local machine, it did not work until I explicitly encapsulated the variable name.

```suggestion
      image: ${MY_REGISTRY}/${MY_USERNAME}/${MY_REPO}:latest
```

Reply:
fixed